packages/compass-aggregations/src/components/pipeline-toolbar/pipeline-header/pipeline-actions.spec.tsx

```diff
@@ -33,6 +37,26 @@ describe('PipelineActions', function () {
       expect(onRunAggregationSpy.firstCall.args).to.be.empty;
     });
 
+    it('export action button', function () {
+      process.env.COMPASS_ENABLE_AGGREGATION_EXPORT = 'true';
+      render(
+        <PipelineActions
+          isOptionsVisible={true}
+          onRunAggregation={onRunAggregationSpy}
+          onToggleOptions={onToggleOptionsSpy}
+          onExportAggregationResults={onExportAggregationResultsSpy}
+        />
+      );
+      const button = screen.getByTestId('pipeline-toolbar-export-aggregation-button');
+      expect(button).to.exist;
+
+      userEvent.click(button);
+
+      expect(onExportAggregationResultsSpy.calledOnce).to.be.true;
+      expect(onExportAggregationResultsSpy.firstCall.args).to.be.empty;
+      delete process.env.COMPASS_ENABLE_AGGREGATION_EXPORT
```

Comment:
This is a super nit since this is just a flag specific to this feature, but it would be more accurate to do this inside an `afterEach` and restore the value present before the test run, rather than delete the variable.